lib/legacySupport.js

```diff
@@ -0,0 +1,14 @@
+var AnalyticsDispatcher = require('./AnalyticsDispatcher');
+var extenders = require('./extenders');
+
+module.exports = function() {
+  Object.keys(extenders).forEach(function (name) {
+    var extender = extenders[name];
+
+    AnalyticsDispatcher.prototype[name] = function () {
+      console.warn('AnalyticsDispatcher.' + name + ' is deprecated, use AnalyticsDispatcher.extend('+name+'(arguments)) instead');
```

Comment:
Add a boolean or something to only warn once (maybe once per extender)? Otherwise it will spam the console.
And disable logging in production

Reply:
how should I detect production?